external/coreclr/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
```

Comment:
I don't care if its in a props file or not.  Originally I had suggested this so that we could quickly do traversals that just read the props without evaluating the entire project.  We aren't doing that, and could always add it later.  The other reason was that folks had to "remember" to put it above the first import when in the project file,  they don't need to do that if it is in a different file.